src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form class="form">
+      <div class="personalInfo">
+        <fieldset class="allFieldset">
+          <legend class="allLegend">
+            Personal information
+          </legend>
+          <div class="personalInput">
```

Comment:
same, why do we need div?

Reply:
initially all code was built on div. But then, in the course of completing the task, I realized that the div does not suit me and  added a label. and div there should have been 4 div for convenient separation of the form sections.